common/src/main/res/values/styles.xml

```diff
@@ -862,13 +931,125 @@
+        <item name="fontFamily">@font/montserrat_semibold</item>
+        <item name="android:textColor">@color/button_text</item>
+        <item name="android:background">@drawable/rounded_ripple_background</item>
+        <item name="android:theme">@style/SecondaryButtonTheme</item>
+        <item name="android:textSize">14sp</item>
+        <item name="android:textAllCaps">false</item>
+        <item name="android:paddingStart">12dp</item>
+        <item name="android:paddingEnd">12dp</item>
+        <item name="android:minHeight">40dp</item>
+        <item name="android:minWidth">72dp</item>
+    </style>
+
+    <style name="Button.Secondary.Small">
+        <item name="android:textSize">13sp</item>
+        <item name="android:minHeight">30dp</item>
+        <item name="android:minWidth">60dp</item>
+        <item name="android:theme">@style/SecondaryButtonTheme.Small</item>
+    </style>
+
+    <style name="Button.Secondary.Small.Blue">
+        <item name="android:theme">@style/SecondaryButtonTheme.Small.Blue</item>
+    </style>
+
+    <style name="SecondaryButtonTheme.Small.Round">
+        <item name="titleTextColor">@color/gray_900</item>
+        <item name="strokeColor">@color/steel_gray_150</item>
+        <item name="cornerRadius">16dp</item>
+    </style>
+
+    <style name="Button.Secondary.Small.Round">
+        <item name="android:paddingStart">0dp</item>
+        <item name="android:paddingEnd">0dp</item>
+        <item name="android:minHeight">20dp</item>
+        <item name="android:minWidth">20dp</item>
+        <item name="android:textSize">10sp</item>
+        <item name="android:theme">@style/SecondaryButtonTheme.Small.Round</item>
+    </style>
+
+
+    <!-- Tertiary -->
+
+    <style name="TertiaryButtonTheme">
+        <item name="cornerRadius">8dp</item>
+        <item name="titleTextColor">@color/gray_900</item>
+        <item name="strokeColor">@android:color/transparent</item>
+        <item name="rippleColor">@color/button_ripple_color</item>
+    </style>
+
+    <style name="TertiaryButtonTheme.Blue">
+        <item name="titleTextColor">@color/dash_blue</item>
+    </style>
+
+    <style name="TertiaryButtonTheme.Red">
+        <item name="titleTextColor">@color/red_300</item>
+    </style>
+
+    <style name="TertiaryButtonTheme.Small">
+        <item name="cornerRadius">6dp</item>
+    </style>
+
+    <style name="TertiaryButtonTheme.ExtraSmall">
+        <item name="cornerRadius">6dp</item>
+    </style>
+
+    <style name="TertiaryButtonTheme.Small.Blue">
+        <item name="titleTextColor">@color/dash_blue</item>
+    </style>
+
+    <style name="TertiaryButtonTheme.Small.Red">
+        <item name="titleTextColor">@color/red_300</item>
+    </style>
+
+    <style name="TertiaryButtonTheme.Large">
+        <item name="cornerRadius">10dp</item>
+    </style>
+
+    <style name="TertiaryButtonTheme.Large.Red">
+        <item name="titleTextColor">@color/red_300</item>
```

Comment:
These changes might cause some merge conflicts with explore-dash. I'll resolve them after one of the branches merged with the `master`.